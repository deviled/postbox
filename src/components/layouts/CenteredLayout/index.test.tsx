import React from 'react';
import { render } from '__tests__/utils';
import { Centered } from '.';

describe('Centered.tsx', () => {
  it('should match snapshot', () => {
    const { baseElement } = render(<Centered />);

    expect(baseElement).toMatchSnapshot();
  });
});
