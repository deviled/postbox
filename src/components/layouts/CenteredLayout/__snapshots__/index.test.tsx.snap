// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Centered.tsx should match snapshot 1`] = `
.c0 {
  width: 100vw;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

<body>
  <div>
    <section
      class="c0"
    />
  </div>
</body>
`;
